.markdownlint-cli2.jsonc

```
@@ -79,7 +79,9 @@
     // link-fragments
     // Disabled as it uses a different heading conversion algo to the TOP website
     "MD051": false
-  },
+    },
+  //Custom rules specific to the project
+  //Docs for each rule can be found in `./markdownlint/docs`
```

The closing brace on line 82 still needs reverting as it was accidentally indented an extra level in a previous commit. Only the comments needed to be moved back with a space after the `//` to be consistent with the other comments.

Comments must share the same indent level as the thing they're attached to, not based on other comments. In this case, the added comments are for the `"customRules"` property so must match its indent level. The other comments are attached to their respective rules and so are indented to match those rules' indent levels.

If it makes it simpler, you can use the "Commit Suggestion" underneath the code suggestion below to commit the necessary changes.
```suggestion
  },
  // Custom rules specific to the project
  // Docs for each rule can be found in `./markdownlint/docs`
```